### PR TITLE
Clarify supported versions

### PR DESCRIPTION
**Pull request by kenny-evitt**
First off, and unrelated to this PR, thanks so much for this project – it's fantastic!

As for this PR, I had a hell of a time getting the simplest bot to visibly* connect/login/spawn to a game. I'm using [Gamocosm](https://gamocosm.com/) to create my Minecraft servers and the only 'version' of the Minecraft server software for which I finally got a bot to connect was "Forge (1.8.8-11.14.4.1581)". Version 1.10.2 and 1.7.10 of the "vanilla" Minecraft server didn't work; nor did Cuberite (whatever version is installed).

I definitely didn't interpret the feature "Supports Minecraft 1.8.9" to imply that that was the *only* version supported. [This comment](https://github.com/PrismarineJS/mineflayer/issues/432#issuecomment-227196657) pretty clearly states exactly that:

> Mineflayer currently support only 1.8.

That comment also outlines what would need to be done to add support for other (older) versions.

But until that happens, some clarification in the README would probably be helpful to other new users.

* I include my looking at the Minecraft server logs as part of this visibility.

### CODE REVIEW

**Comment by roblabla** [comment]
Hey, thanks for the PR.

**TLDR**: I'd prefer replacing the line about versions to say "Supports minecraft from 1.8.0 to 1.8.9", without any asterix or w/e. Makes it nice and explicit which versions we support on a single line. 

Mineflayer is currently on life support really, given that the core developers sadly don't have enough time to dedicate to it. If you would like to help work on it to help bring it up to date, you should really stop by our gitter channel (https://gitter.im/PrismarineJS/mineflayer) and we'd be happy to help with anything you need :).

So about versions. Node-Minecraft-Protocol (NMP) (which is the low-level library that handles all the packet and networking stuff) actually supports multiple versions nowadays.

Whenever we update the Prismarine-Chunk to support 1.9-10-11, mineflayer should start to really be multi-versioned, since all the work to be make it support other versions have already been done for the other components (I think. We never actually had a chance to test it ^^'). I believe prismarine-chunk to be the last outdated piece, but I may be wrong.

So in the meantime, having a warning that we are only compatible with 1.8 seems reasonable. However, I think instead of the current wording, we should just have a single bullet point saying "Supports minecraft from 1.8.0 to 1.8.9", without any asterix. If you could do that change, I'll merge that.

That turned out to be a long post. Sorry :D

**Comment by roblabla** [comment]
Oh and a small note. 1.8.0 to 1.8.9 are all the same protocol version with only extremely minor differences, which is why we support all of those :). It's pretty much 100% sure that it works across all of them, and if there's a broken version, it's probably a simple enough fix.

**Comment by rom1504** [comment]
mineflayer now supports 1.8 -> 1.11
